evm/p2p/lightchain.py

```diff
@@ -325,7 +325,7 @@ def _test():
         network_id=NETWORK_ID,
     )
 
-    chaindb = ChainDB(LevelDB(args.db))
+    chaindb = AsyncChainDB(LevelDB(args.db))
```

Comment:
What do you think about removing this whole `_test` section and relying on just the `trinity` CLI.  With a bit of added CLI functionality you should be able to add the equivalent functionality for the `LocalGethPeerPool` via standard CLI parameters.

Reply:
Makes sense, I'll do that